gpu_support/cuda_utils/test_cuda.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
This script can be greatly simplified, just load `CUDA-Samples` and see does `deviceQuery` run

Reply:
We can expand the testing from there.